public_html/wp-content/plugins/wordcamp-payments/includes/sponsor-invoice.php

```diff
@@ -243,12 +245,16 @@ function required_fields_complete( $submitted_values, $required_fields ) {
 /**
  * Render the Status metabox
  *
- * @param \WP_Post $post
+ * @param WP_Post $post The invoice post
  */
 function render_status_metabox( $post ) {
+	require_once( WP_PLUGIN_DIR . '/wcpt/wcpt-event/class-event-loader.php' );
```

Comment:
The wcpt loader only runs on the wcpt screen, rather than all screens. If this line is removed, then you get a fatal because the base class that WordCamp_Loader extends won't exist.